BraveRewardsUI/Tipping/TippingOverviewView.swift

```diff
@@ -29,14 +29,17 @@ class TippingOverviewView: UIView {
   
   let heartsImageView = UIImageView(image: UIImage(frameworkResourceNamed: "hearts"))
   
-  let faviconImageView = UIImageView().then {
+  let faviconImageView = PublisherIconCircleImageView(size: UX.faviconSize.width, inset: 14).then {
     $0.backgroundColor = UX.faviconBackgroundColor
-    $0.clipsToBounds = true
-    $0.layer.cornerRadius = UX.faviconSize.width / 2.0
     $0.layer.borderColor = UIColor.white.cgColor
     $0.layer.borderWidth = 2.0
   }
   
+  let publisherNameLabel = UILabel().then {
+    $0.appearanceTextColor = .white
+    $0.font = .systemFont(ofSize: 20.0, weight: .medium)
```

Comment:
What if the name is very long, should we shrink it or break to few lines, have you tested it?
How do we do it for other places with pub names?

Reply:
I thought about it, in panel we make it have 2 lines but I think everywhere else it is truncated

Reply:
On Android it truncates the pub name like this
![image](https://user-images.githubusercontent.com/17010094/67711760-09b9e100-f999-11e9-912f-6b76d236a0cb.png)

So probably follow the same. Desktop has bigger text area so it shows on a new line

Reply:
Do you want to add it @kylehickinson?
There's method for truncating strings in StringExtensions

Reply:
It will truncate automatically as its constrained to the right side, no need to truncate manually :)

Reply:
![Simulator Screen Shot - iPhone 11 Pro - 2019-10-28 at 15 43 33](https://user-images.githubusercontent.com/529104/67712129-c318b680-f999-11e9-8606-2b825f5e318a.png)